shared/util/quit-helper.desktop.js

```diff
@@ -47,7 +56,8 @@ function _executeActionsFromRenderer (actions: Array<Action>) {
   ipcRenderer.send('executeActions', actions)
 }
 
-export function executeActions (actions: Array<Action>) {
+export function executeActionsForContext (context: Context) {
```

Comment:
I disagree with this not making the code simpler because it would be decoupling parts.

but it's also not a big change, so it's fine for now